pod2man.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+
+pod2man_helper() {
+    local podfile="${1?}"
+    local manfile="${2?}"
+    local outpath="${3?}"
+    local center="${4?}"
+    pod2man --section 8 --release \
+            --center "${center?}" \
+            --name "${manfile?}" \
+            "${podfile?}" "${outpath?}/share/man/${manfile?}.8"
+}
+
+main() {
+    local outpath="${1?}"
+    mkdir -p "${outpath?}/share/man"
+    type -p pod2man &> /dev/null
+    pod2man_helper "man/puppet-db.pod" "puppet-db" \
+                   "${outpath?}" "manages PuppetDB administrative tasks"
+    pod2man_helper "man/puppet-query.pod" "puppet-query" \
+                   "${outpath?}" "queries PuppetDB data"
+    pod2man_helper "man/puppetdb_conf.pod" "puppetdb_conf" \
+                   "${outpath?}" "PuppetDB CLI configuration"
+}
+
+DIR="$(dirname "$0")"
+main "${1:-"${DIR?}"/target/debug}"
```

Comment:
If we want a main, I'd be inclined to let main handle the args itself, a la: https://google.github.io/styleguide/shell.xml?showone=main#main

Though I might be more inclined to do something like this:

```sh
srcdir="$1"
generate-manpages "$srcdir/target/debug"
```

(Assuming "srcdir" is actually an appropriate name for $1.)